trackman/templates/library/index.html

```diff
@@ -18,6 +18,14 @@ <h1>Library</h1>
     <li class="nav-item"><a class="nav-link" href="{{ url_for('trackman_library.fixup') }}">Fixup</a></li>
 </ul>
 
+<!-- TODO beautify -->
+<h2>search<h2>
+    <p>Only searches exact matches</p>
+    <form action=artist>
```

Comment:
Please quote this and use the url_for Jinja2 filter instead. Also, are you sure this works? This should send a POST request and that should be rejected without a CSRF token.

Reply:
works when patched in in the browser. We should use GET, though.